### ISSUE TITLE
Layer Boundary Size

### ISSUE THREAD

**Issue by Barofar** [glimpse-editor/Glimpse#261]
Hello,
The worst thing in original GIMP is a thing that you never gonna see in any other image editor - manual layer boundary resizing. It slows down the work process a lot!
Normally layer size just automatically adjusts to non-alpha layer content.
But in GIMP every time after I moved a layer, or transformed it, or anything else like this, I have to waste my time for layer resizing. That's annoying a lot.
Of course, I think some people can get some profit from ability to set layer size manually, but I don't think that they should do it by default.
There is even no option to 'Set layer size equal to non-alpha layer content'. You can only set it equal to image size.
I REALLY HOPE that I'll see the dynamic layer size feature in Glimpse! Please..

**Comment by TrechNex** [comment]
Are the new layer-editing features in GNU Image Manipulation Program 2.10.14 sufficient for your needs? If so, we are expecting to base on that for Glimpse 0.2.0 :)

 If not, we'll keep it as UI rewrite candidate but could bring in as a "fork candidate" if we eventually gain the expertise to implement that change.

**Comment by Barofar** [comment]
Well. I really didn't know about that update. I tested it out and... Nope. They made a bit different thing.
![Снимок экрана от 2020-01-05 03-38-31](https://user-images.githubusercontent.com/59454639/71773414-c4ec9080-2f6d-11ea-9556-42434553a3e6.png)
As you can see, now I'm just able to see and edit something outside the canvas (orange dashed line), but I still can't paint outside the layer borders (yellow dashed line). I need to edit that borders first and then paint.
For example in Krita I don't need to edit layer borders. That makes painting process faster. That's how layers works in any image editor but GIMP.
I guess, Krita has such thing as layer borders as well, but they changes automatically depending on layer content. That's the feature I want to see in Glimpse.

**Comment by Member1221** [comment]
Yeah, the weird layer boundaries is something we *arent* keeping in the rewrite, as it’s a pretty big annoyance in general.
